tests/conftest.py

```diff
@@ -57,12 +42,15 @@ def collect(self):
         test_dir = os.path.dirname(str(self.fspath))
         for platform, archs in spec["Output Files"].items():
             for arch, filename in archs.items():
-                # TODO only test Windows PE files for now
-                _, fileext = os.path.splitext(filename)
-                if fileext == ".exe":
-                    filepath = os.path.join(test_dir, "bin", filename)
-                    if os.path.exists(filepath):
-                        yield FLOSSTest(self, platform, arch, os.path.join("bin", filename), spec)
+                filepath = os.path.join(test_dir, filename)
+                if os.path.exists(filepath) and footer.has_footer(filepath):
```

Comment:
Do we want to fallback to read strings from test.yml file if the sample has no appended test data?